docs/source/using/basic-usage.adoc

```diff
@@ -31,6 +31,8 @@ xref:../openshift/openshift-client-binary.adoc#openshift-client-binary-overview[
 For more details about using {project} to manage your local OpenShift cluster, see the xref:../openshift/index.adoc#[Interacting with OpenShift] section.
 
 [[diagram-minishift-architecture]]
+
+//DOC_TAR_MARKER_TITLE_Minishift architecture_IMG_NAME_minishift-architecture_ENDS
```

Comment:
@hferentschik , Please let me know if you have any other idea to tackle this issue. I could come up with this idea.

Reply:
I don't know what you are trying to do here to be honest